src/components/BackgroundCurved.js

```diff
@@ -0,0 +1,58 @@
+import { Box } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const useStyles = makeStyles((theme) => ({
+  wrapper: {
+    position: 'absolute',
+    left: 0,
+    right: 0,
+    bottom: '50%',
+    zIndex: 0,
+    top: 0,
+    margin: '0 auto',
+    background: 'linear-gradient(180deg, #48B2B7 0%, #06FC9D 100%)',
```

Comment:
Could be cool to get the colors from somewhere where they have been defined, probably here: https://github.com/CirclesUBI/circles-myxogastria/blob/main/src/styles/theme.js#L30

Reply:
Similar actually with the `zIndex`, its nice when they are all defined in one place.